paddle/framework/tensor.h

```diff
@@ -78,6 +78,9 @@ class Tensor {
   template <typename T>
   inline T* mutable_data(DDim dims, platform::Place place);
 
+  /*! Size of a single element in data() */
+  inline size_t element_size() const { return holder_->element_size(); }
```

Comment:
Actually, I think there is no need to add this function. A function like `Variable::IsType()` is more useful: https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/framework/variable.h#L41